setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import find_packages, setup
+
+install_requires = [
+    "pandas>=1.2.5,<2.0",
+    "boto3<=1.26.55",
+    "aws-cdk-lib>=2.0.0",
+    "aws-cdk.aws-codestar-alpha>=2.0.0alpha1",
+    "aws-cdk.aws-batch-alpha>=2.0.0alpha1",
```

Comment:
I'm not sure if alpha1 means it's not official yet. If so, shall we instead depend on packages that are stable?

Reply:
Unfortunately the `JobDefinitionContainer` (used to specify parameters such as `shm_size`) is only available in this package for cdk v2. I think it's still under [active development](https://github.com/aws/aws-cdk/blob/main/README.md) at this time, shall we follow up on this?